openapi/generator/cleanFiles.js

```diff
@@ -2,8 +2,12 @@ let fs = require("fs")
 
 let filesJson = fs.readFileSync("./generator/createdFiles.json")
 
-for ( let file of JSON.parse(filesJson) ) {
-  fs.unlinkSync("../"+file.dest)
+try {
+  for (let file of JSON.parse(filesJson)) {
+    fs.unlinkSync("../" + file.dest)
+  }
+} catch (e) {
+
```

Comment:
Empty catch block.
I'm not sure how this is typically handled in JS, but there could be other file system problems other then the files missing.  My only suggestion is add a trace logger, but that seems like it would get missed to?

Reply:
This try catch is for the sole purpose of an invalid createdFiles.json... this happens when the build fails causing the next build to fail fully because of the json file